frontend/src/components/MainToolbar.vue

```diff
@@ -153,6 +154,11 @@ export default {
     ]),
     handleLogout () {
       this.$auth.signout()
+    },
+    showInfoDialog () {
+      this.$refs.infoDialog.showDialog()
+      this.$refs.infoDialog.fetchVersions()
```

Comment:
Why the parent component has to call methods on the child in order to initialize it?